services/movie.py

```diff
@@ -28,13 +29,15 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
 ) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
-
-    return movie
+    with transation.atomic():
```

Comment:
There is a typo in the `transaction.atomic()` context manager. It should be `transaction.atomic()` instead of `transation.atomic()`.